src/porepy/models/material_constants.py

```diff
@@ -327,6 +327,7 @@ def default_constants(self) -> dict[str, number]:
 
         """
         # Default values, sorted alphabetically
+        # TODO: Numerical method parameters may find a better home soon.
```

Comment:
Agree! I can think of a few reasons for revisiting this:
1. Cleanliness.
2. Getting an overview of such numerical parameters in use.
3. (hopefully) introduce scaling of numerical parameters, just as with physical ones. I know that @jwboth has started thinking of this.

@IvarStefansson: Should we make this a user project?

For now, I think this is a fair placement, so no need for action at the moment.

Reply:
Thanks for the hint, @keileg. I plan to raise this issue soon and would suggest a PR based on some experiences I am gathering right now. Not scaling particularly the characteristic tolerance has led to big inconsistencies in my examples.

Reply:
I'll create an issue. Feel free to expand it later, @jwboth.